src/test/java/core/basesyntax/service/impl/FileReaderImplTest.java

```diff
@@ -0,0 +1,49 @@
+package core.basesyntax.service.impl;
+
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class FileReaderImplTest {
+    private static final String VALID_PATH = "src/test/resources/inputFile.csv";
+    private static final String INVALID_PATH = "src/test/resources/wrong.csv";
+    private static final String EMPTY_FILE = "src/test/resources/inputEmptyFile.csv";
+    private static FileReaderImpl fileReader;
+    private static List<String> inputData;
+
+    @BeforeClass
+    public static void beforeClass() {
+        fileReader = new FileReaderImpl();
+        inputData = new ArrayList<>();
+        inputData.add("type,fruit,quantity");
+        inputData.add("b,banana,20");
+        inputData.add("b,apple,100");
+        inputData.add("s,banana,100");
+        inputData.add("p,banana,13");
+        inputData.add("r,apple,10");
+        inputData.add("p,apple,20");
+        inputData.add("p,banana,5");
+        inputData.add("s,banana,50");
+    }
+
+    @Test
+    public void readFromFile_isOk() {
```

Comment:
давай зробимо назви тестових методів консистентними. Вони повинні складатися або з двох або з трьох частин (якщо обереш з трьох то всі тестові методи повинні складатися з трьох частин)
```suggestion
    public void readFromFile_ok() {
```